examples/plot_discriminant_scores.py

```diff
@@ -81,12 +43,41 @@
 )
 
 # Add the histograms
-plot_histo.add(hist_dips_light, reference=True)
-plot_histo.add(hist_dips_c, reference=True)
-plot_histo.add(hist_dips_b, reference=True)
-plot_histo.add(hist_rnnip_light)
-plot_histo.add(hist_rnnip_c)
-plot_histo.add(hist_rnnip_b)
+for tagger, linestyle in zip(taggers, linestyles):
+
+    plot_histo.add(
+        Histogram(
+            df[is_light][f"disc_{tagger}"],
+            label="Light-flavour jets" if tagger == "dips" else None,
+            colour=flav_cat["ujets"]["colour"],
+            ratio_group="ujets",
+            linestyle=linestyle,
+        ),
+        reference=tagger == "dips",
+    )
+    plot_histo.add(
+        Histogram(
+            df[is_c][f"disc_{tagger}"],
+            label="$c$-jets" if tagger == "dips" else None,
+            colour=flav_cat["cjets"]["colour"],
+            ratio_group="cjets",
+            linestyle=linestyle,
+        ),
+        reference=tagger == "dips",
+    )
+    plot_histo.add(
+        Histogram(
+            df[is_b][f"disc_{tagger}"],
+            label="$b$-jets" if tagger == "dips" else None,
+            colour=flav_cat["bjets"]["colour"],
+            ratio_group="bjets",
+            linestyle=linestyle,
+        ),
+        reference=tagger == "dips",
+    )
 
 plot_histo.draw()
+plot_histo.make_linestyle_legend(
```

Comment:
alos here, would be good to explain it in the examples

Reply:
I added a comment